### PR TITLE
internal/splunk: fallback to host if cloud resource id not found

### PR DESCRIPTION
**Pull request by jrcamp**
APM correlation will need to do same thing so add fallback logic in common.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/client9/misspell v0.3.4
 	github.com/go-lintpack/lintpack v0.5.2 // indirect
 	github.com/golangci/golangci-lint v1.31.0
-	github.com/google/addlicense v0.0.0-20200622132530-df58acafd6d5
+	github.com/google/addlicense v0.0.0-20200906110928-a0294312aa76
```
Comment (bogdandrutu) [inline go.mod:9]: Is this needed in this PR?

**Review by bogdandrutu — COMMENTED** [review]
splunk is the only vendor specific thing in common anymore, would you consider moving that to internal/splunk please?

**Review by dmitryax — APPROVED** [review]
LGTM

**Comment by jrcamp** [comment]
> splunk is the only vendor specific thing in common anymore, would you consider moving that to internal/splunk please?

I will do in followup PR. I should make internal/splunk its own go module I assume?

**Comment by tigrannajaryan** [comment]
Build failed.

**Comment by bogdandrutu** [comment]
Please rebase

**Comment by jrcamp** [comment]
@tigrannajaryan fixed

**Pull request merged**